File: example-application/src/test/java/com/example/application/ExampleResourceTest.java
package com.example.application;

import io.quarkus.test.common.http.TestHTTPEndpoint;
import io.quarkus.test.junit.QuarkusTest;
import org.junit.jupiter.api.Test;

import static io.restassured.RestAssured.when;
import static org.hamcrest.Matchers.*;

@QuarkusTest
@TestHTTPEndpoint(ExampleResource.class)
class ExampleResourceTest {

    @Test
    void testGet() {
        when().get()
                .then().statusCode(200)
                .body(is(not(emptyString())));
    }

}

File: example-application/src/test/kotlin/com/example/application/ExampleResourceTest.kt
package com.example.application

import io.quarkus.test.common.http.TestHTTPEndpoint
import io.quarkus.test.junit.QuarkusTest
import io.restassured.RestAssured
import org.hamcrest.Matchers
import org.junit.jupiter.api.Test

@QuarkusTest
@TestHTTPEndpoint(ExampleResource::class)
internal class ExampleResourceTest {
  @Test
  fun testGet() {
    RestAssured.`when`().get()
      .then().statusCode(200)
      .body(Matchers.`is`(Matchers.not(Matchers.emptyString())))
  }
}